### ISSUE TITLE
Add `pr-opened` label to issues when a pr is opened that fixes/refs the issue

### ISSUE THREAD

**Issue by hzoo** [babel/babel-bot#5]
name could be different just a suggestion

Parse

1. just the "fixed tickets" line in our pr template
2. anything that looks and smells like github ID # and is an issue
3. (1 || 2) + title of the ticket

**Comment by DrewML** [comment]
Item 1 in the list above [is done](https://github.com/babel/babel-bot/blob/a36ba80/src/handlers/pull_request/opened.js), but I'm hitting a _super_ weird issue where the sha for the webhook (what we use for the auth) doesn't match what I'm getting for the payload + secret. Haven't had this problem with any other type of events.

**Comment by marcoslhc** [comment]
@DrewML Ihad a similar issue sometime ago. IDK if it helps: http://stackoverflow.com/questions/30269030/github-push-event-signature-dont-match

**Comment by DrewML** [comment]
@marcoslhc you rock - thank you! That's probably the issue. Going to give it a shot tonight.

**Comment by hzoo** [comment]
Done in https://github.com/babel/babel-bot/commit/a36ba808d1ae75e0342c7de1056729d16af70db5 already? I already saw it in action lol

<img width="413" alt="screen shot 2017-01-14 at 6 20 34 pm" src="https://cloud.githubusercontent.com/assets/588473/21958891/291776f0-da86-11e6-9b2c-5f8121fc53ba.png">

**Comment by DrewML** [comment]
@hzoo Yeah, I had temporarily disabled signature validation and forgot to re-enable it, lol. Going to put in the fix @marcoslhc recommended now.

**Comment by DrewML** [comment]
Done so far:

1. `just the "fixed tickets" line in our pr template`
